packages/flutter/test/widgets/modal_barrier_test.dart

```diff
@@ -77,7 +78,32 @@ void main() {
     await tester.pump(const Duration(seconds: 1)); // end transition
 
     // Tap on the barrier to dismiss it
-    await tester.tap(find.byKey(const ValueKey<String>('barrier')));
+    await tester.press(find.byKey(const ValueKey<String>('barrier')));
```

Comment:
Can you keep the existing test to ensure that this continues to work as is and add a new test instead?